src/WebJobs.Extensions.DurableTask.Analyzers/SyntaxNodeUtils.cs

```diff
@@ -312,7 +313,7 @@ internal static string GetQualifiedTypeName(ITypeSymbol typeInfo)
                     var tupleUnderlyingType = namedTypeInfo.TupleUnderlyingType;
                     if (tupleUnderlyingType != null)
                     {
-                        return $"System.Tuple<{string.Join(", ", tupleUnderlyingType.TypeArguments.Select(x => x.ToString()))}>";
+                        return $"({string.Join(", ", tupleUnderlyingType.TypeArguments.Select(x => x.ToString()))})";
```

Comment:
Is the goal of this to return a type name tuples that is valid for both `Tuple` and `ValueTuple`?

Reply:
Actually no. This fixes a bug with Tuples/ValueTuples and also covers a new test case that came up. Tuples (and usually ValueTuples) will actually bypass this check and go to the line
`return typeInfo.ToString();`

Previously I was treating Tuples and ValueTuples as the same thing. I realized my mistake and this change fixes a case where the ITypeSymbol for a ValueTuple will be something like "(string jobId, int batchNumber)" instead of "(string, int)"

This happens in this case (this example omits the function definition, you can assume it takes a (string, int) ValueTuple):
```
 var (jobId, batchNumber) = context.GetInput<(string, int)>();

var rows = await context.CallActivityAsync<IEnumerable<(string, string)>>("GetRowsFromBatch", (jobId, batchNumber));
```

GetQualifiedTypeName is basically a ToStringInAStandardWay method. It is usually not used in the step to compare types and is usually used only when creating the message to display to the user. But in this case, I actually need to get a "different version" (TupleUnderlyingType) of the INamedTypeSymbol to be able to compare them. This particular ValueTuple situation is the only one that I ran into where I needed to use this.

Previously I didn't understand Tuples and thought that Tuples and (string, int) (ValueTuples) were the same thing. They are not and I was incorrectly comparing them.